xbmc/windowing/windows/WinEventsWin32.cpp

```diff
@@ -665,9 +665,18 @@ LRESULT CALLBACK CWinEventsWin32::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, L
         else
         {
           // API call such as SetWindowPos or SwapChain->SetFullscreenState
-          newEvent.type = XBMC_VIDEORESIZE;
-          newEvent.resize.w = g_sizeMoveWidth;
-          newEvent.resize.h = g_sizeMoveHight;
+          if (DX::Windowing()->IsFullScreen())
+          {
+            newEvent.type = XBMC_FULLSCREEN_UPDATE;
+            newEvent.resize.w = DX::Windowing()->GetWidth();
```

Comment:
I'm not sure if use `DX::Windowing()->GetWidth()` is correct here because is supposed this event is triggered by OS and can tell new window size (i.e. resolution changed externally). Then seems correct use same g_sizeMoveWidth / Height as is based on event data:

```c++
      g_sizeMoveWidth = GET_X_LPARAM(lParam);
      g_sizeMoveHight = GET_Y_LPARAM(lParam);
```

Reply:
I have the same doubts because it depends on the sequence of Windows events and how Kodi updates the various internal states.

But I also have doubts about the WM_SIZE parameters and don't think they provide an equivalent. The #ifdef exists in the XBMC_VIDEORESIZE handler for the purpose of using the desktop resolution information instead of newEvent.resize.w, newEvent.resize.h (which contain the WM_SIZE parameters)

With win 10 22h2 I am unable to find the use case that required the code of the ifdef, it could be related to dpi change with win < 10 fcu, win 8 and older (see other comment).